pyupgrade/_plugins/lru_cache.py

```diff
@@ -21,6 +24,23 @@ def _remove_call(i: int, tokens: List[Token]) -> None:
     del tokens[i:j + 1]
 
 
+def _replace_lru_cache_with_cache_import(
+        i: int, tokens: List[Token], imports: Dict[Any, Any] = {},
+) -> None:
+    _replace_lru_cache_with_cache(i, tokens)
+
+    if 'cache' not in imports['functools']:
+        n = find_token(tokens, 0, 'lru_cache')
+        tokens[n] = Token(name='NAME', src='cache, lru_cache')
+        imports['functools'].add('cache')
+
+
+def _replace_lru_cache_with_cache(i: int, tokens: List[Token]) -> None:
+    j = find_token(tokens, i, 'lru_cache')
+    tokens[j] = Token(name='NAME', src='cache')
+    del tokens[j + 1:j + 6]
```

Comment:
this is error prone, it could be `@functools.lru_cache(     max_size    = None   )` -- use the `find_and_replace_call` helper